crates/bevy_ui/src/lib.rs

```diff
@@ -83,7 +83,7 @@ impl Plugin for UiPlugin {
             .add_system_to_stage(
                 CoreStage::PostUpdate,
                 widget::text_system
-                    .before(UiSystem::Flex)
+                    .after(UiSystem::Flex)
```

Comment:
as said on discord, this is wrong, this system needs to run before flex (and maybe after too)